_posts/2019-02-05-configurable-builds-part-1.md

```diff
@@ -159,12 +163,12 @@ java_library(
 
 If you use `--cpu` and `crosstool_top` to define what `:android` means, what
 happens when an app supports a new Android phone with a different CPU?
-`:android` won't trigger, the app won't get required Android support libraries,
-and it will crash.
+`:android` won't trigger because `--cpu` no longer matches, the app won't get
+required Android support libraries, and it will crash.
```

Comment:
Change "will crash" into "won't work properly". We can only hope the problem would be as obvious as a crash.

Reply:
How about "break"?  I'm trying to go for dramatic effect, show tightly that these obscure scenarios you might not consider can have unpleasantly surprising effects.

Reply:
"break" sounds good.

Reply:
Done.